src/utils/wallet.js

```diff
@@ -304,6 +304,18 @@ const wallet = {
       tokens: dataToken,
     }));
 
+    // before cleaning data, check if we need to transfer xpubkey to wallet:accessData
+    const accessData = hathorLib.wallet.getWalletAccessData();
+    if (accessData.xpubkey === undefined) {
+      // XXX from v0.12.0 to v0.13.0, xpubkey changes from wallet:data to access:data.
+      // That's not a problem if wallet is being initialized. However, if it's already
+      // initialized, we need to set the xpubkey in the correct place.
+      const oldData = JSON.parse(localStorage.getItem('wallet:data'));
```

Comment:
What if `wallet:data` is undefined? Have you tested initializing a brand new wallet?

Reply:
This situation cannot happen. Only chance `wallet:data` is undefined in `localStorage` is if the user already has the newest version. In that case, he wouldn't enter this `if`.

Yes, I tested initializing a new wallet.

Reply:
Ok. Thanks!